jetson/percep/main.cpp

```diff
@@ -94,7 +92,9 @@ int main() {
     #if WRITE_CURR_FRAME_TO_DISK && AR_DETECTION && OBSTACLE_DETECTION
       if (iterations % FRAME_WRITE_INTERVAL == 0) {
         Mat rgb_copy = src.clone(), depth_copy = depth_img.clone();
-        cerr << "Copied correctly" << endl;
+        #if PERCEPTION_DEBUG
+          cout << "Copied correctly" << endl;
```

Comment:
Can you make sure there are 4 spaces in tab? It looks like only two?

Reply:
FIxed